### DISCUSSION TITLE
Domotizzare impianto riscaldamento condominiale

### DISCUSSION THREAD

**Discussion by luky956** [indomus/forum#2722] [Hardware & Firmware]
Ciao a tutti, 

vi chiedo aiuto per capire quale sia la soluzione migliore per domotizzare un impiato di riscaldamento, si tratta di un impianto a pavimento in un condominio con riscaldamento centralizzato; nel mio appartamento c'è un cronotermostato che l'idraulico definisce come "pilota", dice che si tratta del comando principale che temporizza il riscaldamento e che porta l'acqua calda all'impianto e alle zone (si tratta di due bagni e un disimpegno) non controllate dagli altri termostati. Questi ultimi sono tre e controllano rispettivamente living/cucina, studio e camera da letto. Mi è stato detto che il cronotermostato principale deve avere un target di temperatura sempre più alto rispetto a quello impostato sui termostati che controllano le varie zone. 
Purtroppo i termostati delle varie zone permetto sono un'accensione manuale, non temporizzata, tramite una rotella sulla quale si imposta la temperatura da raggiungere.
Le mie competenze sono purtroppo limitate, l'idraulico mi dice che andrebbero montati 4 termostati smart per poter controllare da remoto, esiste qualche alternativa per l'integrazione in home assistant? Si potrebbe fare con Shelly abbinati a sensori di temperatura zigbee? 

Spero di essere stato conprensibile, grazie a chi vorrà anche solo tentare di darmi una mano!

**Comment by Bergoglio** [comment]
Vediamo se ho capito :
1) il cronotermostato principale apre una valvola generale di prelievo acqua dalla colonna condominiale per tutto il tuo appartamento
2) i termostati nelle zone aprono delle valvole sull'impianto interno al tuo appartamento per alimentare la zona specificatamente comandata.
Se è così la domotizzazione è alquanto semplice (come concetti)

Domande pratiche per scegliere la strada da percorrere :
1) hai già una installazione Home Assistant funzionante ?
2) hai già dei sensori temperatura + umidità funzionanti in Home Assistant ? Quali ?
3) hai in mente di fare dei lavori per tirare dei cavi nuovi / aggiuntivi o vuoi passare tutto in wireless ?
4) se passi o vuoi passare in wireless, che protocollo usi o vuoi usare ? Personalmente preferisco il wi-fi classico ed il LoRa
5) le elettrovalvole che comandano la presa generale + le zone sono radunate tutte insieme o sono sparse per la casa ?

**Reply by luky956** [reply]
Ciao, innanzitutto ti ringrazio per la risposta.

L’impianto è esattamente come l’hai descritto tu, per tutto il resto:

1. Ho un’installazione home assistant che gira su un mini pc collegato in lan, con chiavetta zigbee.
2. Per ora gli unici sensori temperatura e umidità sono quelli che espongono i condizionatori, poco affidabili. Posso valutare termostati con sensori incorporati oppure sensori esterni da posizionare in casa.
3. Vorrei restare in wireless. Wi-Fi o zigbee sarebbero le scelte migliori 
5. Il gruppo valvole è in tutto in un’unica posizione, facilmente ispezionabile.

**Comment by Bergoglio** [comment]
Allora, zigbee non lo uso ma non lo sconsiglio; con zigbee ci farei girare i sensori temperatura + umidità. Bisogna solo trovare dei sensori che non siano "pigri" ovvero che rilevino con un'alta frequenza e non ogni tanto; da questo punto di vista i sensori wifi che ho trovato non vanno bene, sono pigri per risparmiare le batterie. Io ho adottato sensori LoRa perché anche se wireless il segnale fora pure i muri di cemento armato, ma costano cari e li ho scelti per un problema di segnale che ho in casa. Valuterei comunque attentamente di acquistare dei multi-sensori (temp+umidità+presenza+luce+qualità aria) perché un domani neanche tanto lontano potresti voler aggiungere delle funzioni.
Per gli attuatori procederei con i classici Shelly (non la linea pro perché hai detto che non vuoi tirare cavi, neanche un cavo lan fino al gruppo valvole)
Come budget dove vuoi rimanere ? Basso, alto, altissimo ?

**Reply by luky956** [reply]
Sul budget non vorrei esagerare, certo è che prefirei spendere qualcosa in più ed avete prodotti validi piuttosto che buttare via soldi in dispositivi scadenti. Quindi proponi di agire direttamente sulle valvole con gli shelly, oppure montarli sui cronotermostati e poi fare l'automazione da home assistant?

**Comment by Bergoglio** [comment]
Gli Shelly possono solo agire solo sulle valvole, non vedo altro modo.
Bisogna solo decidere se montarli al posto dei cronotermostati o direttamente vicino alle elettrovalvole. Comunque io parto dall'assunto di eliminare i cronotermostati per ricreare e gestire le logiche all'interno di Home Assistant.

Iniziamo a definire chi esegue che cosa :
1) ci sono comunque dei lavori elettrici da fare, chi li effettuerà ? Tu o un elettricista ? Non sono lavori complicati ma un minimo di manualità bisogna averla.
2) quasi sicuramente qualche piccolo intervento non elettrico tipo una scatola da smurare per installarne una più grande etc. salterà fuori. Anche qui : chi li esegue ?
3) se si vuole valutare piccolissime modifiche più meccaniche che idrauliche tipo montare la motorizzazione di una valvola, chi esegue ?

**Comment by luky956** [comment]
Allora penso che la cosa più semplice da fare sia montarli all’interno delle 503 dei cronotermostati. La parte elettrica e di configurazione di home assistant possa farla io, opere murarie o idrauliche/meccaniche meglio farle fare ad un professionista.A questo punto immagino che poi si farà una automazione home assistant che lavorerà su fasce orarie e avrà come trigger i valori esposti dai sensori, corretto?

**Reply by Bergoglio** [reply]
La strada software è più o meno quella.

Visto che vai di fai da te elettrico ti do due dritte d'esperienza :
1) gli shelly hanno morsetti su cui al massimo ci infili un 1,5 mm2
2) per far entrare i cavi in quei morsetti è meglio se usi i puntalini come i tedeschi, altrimenti preparati a bestemmiare. Hai la pinza per crimpare i puntalini ?

Per il collegamento degli shelly nelle 503 dei cronotermostati, arriva l'alimentazione in quelle scatole (220 v o 12 / 24 v corrente continua) ? Controlla, ma dubito fortemente. Occhio che parlo di alimentazione, non cavi per attivare l'elettrovalvola.
